SyntaxError: Unexpected token F in JSON at position 0

getting:
      
      I20210125-20:46:40.823(10)? SyntaxError: Unexpected token F in JSON at position 0
      I20210125-20:46:40.823(10)?     at JSON.parse (<anonymous>)
      I20210125-20:46:40.823(10)?     at B.json (....../node_modules/mailgun.js/dist/mailgun.js:2:76174)
      I20210125-20:46:40.823(10)?     at processTicksAndRejections (internal/process/task_queues.js:97:5)

in this simple code:

        const formData = require('form-data');
        const Mailgun = require('mailgun.js');
        const mailgun = new Mailgun(formData);
        const mg = mailgun.client({username: 'api', key:'xxxxxx-xxxx-xxx' });
        mg.messages.create('domain.com', {
            from: "contact <contacto@domain.com>",
            to: ["info@domain.com"],
            subject: "Hello",
            text: "Testing some Mailgun awesomness!",
            html: "<h1>Testing some Mailgun awesomness!</h1>"
        })
            .then(msg => console.log(msg)) // logs response data
            .catch(err => console.log(err)); // logs any error



verions: 3.0.0
node: v14.3.0
I am using meteor as framework

Worth noting i've got

```
[ERROR] SyntaxError: Unexpected token F in JSON at position 0
    at JSON.parse (<anonymous>)
    at B.json (/Users/DVF/Projects/deedmob/www/node_modules/mailgun.js/dist/webpack:/mailgun/node_modules/node-fetch/dist/index.cjs:313:15)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,5 @@
 node_modules/
 results.xml
 build
+
+./test.js

--- a/lib/error.ts
+++ b/lib/error.ts
@@ -4,7 +4,6 @@ import APIErrorOptions from './interfaces/APIErrorOptions';
 export default  class APIError extends Error {
   status: number | string;
   stack: string;
-  details: string;
 
   constructor({
     status,
@@ -12,12 +11,11 @@ export default  class APIError extends Error {
     message,
     body = {}
   }: APIErrorOptions) {
-    const { message: bodyMessage, error } = body;
+    const { error } = body;
     super();
 
     this.stack = null;
     this.status = status;
     this.message = message || error || statusText;
-    this.details = bodyMessage;
   }
 }

--- a/lib/request.ts
+++ b/lib/request.ts
@@ -75,8 +75,7 @@ class Request {
     if (!response?.ok) {
       throw new APIError({
         status: response?.status,
-        statusText: response?.statusText,
-        body: await response?.json()
+        statusText: response?.statusText
       } as APIErrorOptions);
     }
 

